.gitignore

```diff
@@ -87,3 +87,8 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+#.DS_Store
+.DS_Store
+/.DS_Store
+/./.DS_Store
```

Comment:
Let's remove this change from this PR since it'll be merged in a different way following changes requested in other PR.